remotes/docker/pusher.go

```diff
@@ -180,11 +184,14 @@ func (p dockerPusher) push(ctx context.Context, desc ocispec.Descriptor, ref str
 				return nil, err
 			}
 
-			if resp.StatusCode == http.StatusUnauthorized {
+			switch resp.StatusCode {
+			case http.StatusUnauthorized:
 				log.G(ctx).Debugf("failed to mount from repository %s", fromRepo)
 
 				resp.Body.Close()
 				resp = nil
+			case http.StatusCreated:
+				mounted = true
```

Comment:
Along the same lines as https://github.com/moby/moby/pull/44963#discussion_r1153788991 (https://github.com/moby/moby/blob/0656059ae7ab1d0b14afd3bd5f4533f2f2083ff0/distribution/push_v2.go#L379), would it make sense for this to also include where it mounted _from_? (perhaps a `string` or ref pointer instead of just a `bool`?  I am not a containerd maintainer though :bow:)

Reply:
Done.